ttg/ttg/parsec/ttg.h

```diff
@@ -339,7 +339,7 @@ namespace ttg_parsec {
       for(int i = 0; i < (int)parsec_nb_devices; i++) {
           parsec_device_module_t *device = parsec_mca_device_get(i);
           if( NULL == device ) continue;
```

Comment:
This is interesting... I'm not saying this is wrong, but I was under the impression that 
``` i == device->device_index```
should be a true invariant... Is there something more fishy? Or my understanding is wrong?

I will check what happens in `parsec_device_cuda_module_init()`